_docs/run.md

```diff
@@ -9,7 +9,16 @@ nav: 3
 You have nothing to consider specially, simply navigate to your Pico install using your favorite web browser. Pico's default contents will explain how to use your brand new, stupidly simple, blazing fast, flat file CMS.
 
 ### You don't have a web server?
-Starting with PHP 5.4 the easiest way to try Pico is using [the built-in web server of PHP][PHPServer]. Please note that PHPs built-in web server is for development and testing purposes only!
+Starting with PHP 5.4 the easiest way to *try* Pico is using PHP's own [built-in web server][PHPServer]. **Please Note** that PHP's built-in web server is for development and testing purposes only!  It is **not** suitable for production use.
+
+* If you decide you like Pico, but need help setting up a more permanent solution, please see our [some section below about server configuration]().
+* Also, only mentioning PHP's internal server seems a bit like a wasted page.  Do we even need a "Run" section?
```

Comment:
It should be sufficient to link to more in-depth pages for Apache, nginx, PHP's built-in webserver and maybe others here...